pkg/fanal/secret/builtin-rules.go

```diff
@@ -743,6 +744,24 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`pscale_tkn_(?i)[a-z0-9\-_\.]{43}`),
 		Keywords: []string{"pscale_tkn_"},
 	},
+	{
+		ID:       "private-packagist-user-token",
+		Category: CategoryPrivatePackagist,
+		Title:    "Private Packagist user token",
+		Severity: "HIGH",
+		// https://packagist.com/docs/composer-authentication#token-format
+		Regex:    MustCompile(`packagist_uut_(?i)[a-z0-9]{68}`),
```

Comment:
> a prefix, a 60 hexadecimal character long random part, and an eight hexadecimal character long checksum

IIUC we can use `a-f0-9`.

Reply:
Good point, not sure why I didn't think of that myself :see_no_evil: